pkg/restapi/oidc/helper.go

```diff
@@ -20,3 +20,12 @@ func openStore(p storage.Provider, name string) (storage.Store, error) {
 
 	return p.OpenStore(name)
 }
+
+// validation rules on the received user claims from the OIDC provider go here
+func evaluateClaims(u *endUser) error {
+	if u.Sub == "" {
```

Comment:
this seems to be a simplistic claims evaluation. Will this be enhanced later? probably add a TODO here

Reply:
@Baha-sk yes - the intent is to put in all validations on the end-user claims received from the OIDC provider here (see comment on line 24)